UT-CostItem-COWI/IR_UT_COWI_Cost.ifc

```diff
@@ -0,0 +1,190 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION((''),'2;1');
+FILE_NAME('','2021-02-01T20:51:32',(''),(''),'buildingSMART IfcKit','IfcFamily','');
+FILE_SCHEMA(('IFC4'));
+ENDSEC;
+
+DATA;
+
+#1= IFCPROJECT('2JbM09ptr5SRkGdojkjLdj',#2,'UT_RSS_2',$,$,$,$,(#3),#4);
+#2= IFCOWNERHISTORY(#9,#10,$,.ADDED.,1606203924,$,$,1606203924);
+#3= IFCGEOMETRICREPRESENTATIONCONTEXT('Model',$,3,1.0E-08,#11,#12);
+#4= IFCUNITASSIGNMENT((#13,#14,#15,#16,#17));
+#9= IFCPERSONANDORGANIZATION(#18,#19,$);
+#10= IFCAPPLICATION(#20,'0.1.4.0','UT_RSS_2 v1.0.0.0','UT_RSS_2 v1.0.0.0');
+#11= IFCAXIS2PLACEMENT3D(#21,#22,#23);
+#12= IFCDIRECTION((0.,1.));
+#13= IFCSIUNIT($,.LENGTHUNIT.,$,.METRE.);
+#14= IFCSIUNIT($,.AREAUNIT.,$,.SQUARE_METRE.);
+#15= IFCSIUNIT($,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#16= IFCSIUNIT($,.PLANEANGLEUNIT.,$,.RADIAN.);
+#17= IFCSIUNIT($,.TIMEUNIT.,$,.SECOND.);
+#18= IFCPERSON('lars','lars',$,$,$,$,$,$);
+#19= IFCORGANIZATION($,'Unknown',$,$,$);
+#20= IFCORGANIZATION($,'Geometry Gym Pty Ltd',$,$,$);
+#21= IFCCARTESIANPOINT((0.,0.,0.));
+#22= IFCDIRECTION((0.,0.,1.));
+#23= IFCDIRECTION((1.,0.,0.));
+
+#5= IFCRELAGGREGATES('22oqctZZ95DRRIKQ5FlGRP',#2,$,$,#1,(#24));
+
+#6= IFCRELASSOCIATESCLASSIFICATION('3ePlM42uPEGxYhmpYQpal0',$,$,$,(#1),#25);
+#25= IFCCLASSIFICATION($,'201802','2018-06','Prosesskode','Standard specification of work for road projects used in Norway; issued in 2 handbooks, R761 (General specification 1) and R762 (General specification 2)','https://www.vegvesen.no/_attachment/2362831/binary/1269863?fast_title=H%C3%A5ndbok+R761+Prosesskode+1+Standard+beskrivelsestekster+for+vegkontrakter.pdf',('.'));
+#26= IFCCLASSIFICATIONREFERENCE($,'72.14','a',#25,$,$);
+
+#7= IFCRELDECLARES('2NTYCyomfC$uyNxNvK2zxD',$,$,$,#1,($,#28));
```

Comment:
A null in a list isn't expected. ($,#28)

Reply:
True. Old file! Thanks. There were some issues initializing sets in ifckit. I fixed that now. will correct.